tiles/src/main/java/com/protomaps/basemap/layers/Earth.java

```diff
@@ -29,6 +30,19 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
     } else if (sourceLayer.equals("ne_10m_land")) {
       features.polygon(this.name()).setZoomRange(5, 5).setBufferPixels(8).setAttr("kind", "earth");
     }
+    if (sourceLayer.equals("ne_10m_glaciated_areas")) {
```

Comment:
This section deserves a comment explaining the problem and why this particular solution works

Reply:
Added one, thanks for the tip!